src/main/java/seedu/address/model/task/Description.java

```diff
@@ -10,8 +10,8 @@
 public class Description implements Comparable<Description> {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Description should not be blank or start with white spaces,"
-                    + " and the length should be less than 256 characters.";
+            "Description should only contain alphanumeric characters and punctuations,"
```

Comment:
Will it be better to say "Description should only contain alphanumeric characters and punctuations, and it has a character limit of 1-255 characters.", cuz I think its a little awkwardly phrased now HAHA